desktop/webpack.config.development.js

```diff
@@ -18,7 +19,8 @@ const defines = {
 console.warn('Injecting dev defines: ', defines)
 
 config.debug = true
-config.devtool = NO_SOURCE_MAPS ? undefined : 'inline-source-map'
+config.cache = true
+config.devtool = NO_SOURCE_MAPS ? undefined : 'eval-source-map'
```

Comment:
Did you intend to change this again? I seem to recall we just changed it due to issues with `eval-source-map` in Chrome.